packages/components/organisms/f-cookie-banner/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@justeat/f-cookie-banner",
+  "description": "Fozzie Cookie Banner – Cookie Banner",
+  "version": "0.1.0",
+  "main": "dist/f-cookie-banner.umd.min.js",
+  "files": [
+    "dist",
+    "test-utils"
+  ],
+  "homepage": "https://github.com/justeat/fozzie-components/tree/master/packages/components/organisms/f-cookie-banner",
+  "contributors": [
+    "Github contributors <https://github.com/justeat/fozzie-components/graphs/contributors>"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:justeat/fozzie-components.git"
+  },
+  "bugs": {
+    "url": "https://github.com/justeat/fozzie-components/issues"
+  },
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=10.0.0"
+  },
+  "keywords": [
+    "fozzie"
+  ],
+  "scripts": {
+    "prepublishOnly": "yarn lint && yarn test && yarn build",
+    "build": "vue-cli-service build --target lib --name f-cookie-banner ./src/index.js",
+    "demo": "vue serve --port 8080 ./src/demo/Index.vue",
+    "lint": "vue-cli-service lint",
+    "lint:fix": "yarn lint --fix",
+    "lint:style": "vue-cli-service lint:style",
+    "test": "vue-cli-service test:unit",
+    "webdriver:delay:chrome": "yarn test:wait-for-server && yarn webdriver:start:chrome",
```

Comment:
```
    "webdriver:delay:chrome": "yarn test:wait-for-server && yarn webdriver:start:chrome",
    "webdriver:start:chrome": "wdio ../../wdio.conf.js"
```

should be removed in favour of

```
    "test-component:chrome": "wdio ../../../../wdio.conf.js --suite component",
    "test-a11y:chrome": "wdio ../../../../wdio.conf.js --suite a11y"
```

This is already part of the generator, but I added this change since this branch was created 👍🏼

Reply:
Done